docker/fasttextv2/Dockerfile

```diff
@@ -0,0 +1,19 @@
+
+FROM demisto/python-deb:2.7.17.4583
+
+COPY requirements.txt .
+
+RUN apt-get update && apt-get install -y zlib1g-dev libicu-dev g++
```

Comment:
Remove this. Add dev libs to the step where we install python-dev and later remove it.